utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -1446,6 +1446,20 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "SandboxIR",
+    srcs = glob([
+        "lib/SandboxIR/*.cpp",
+    ]),
+    hdrs = glob(["include/llvm/SandboxIR/*.h"]) +
+    ["include/llvm/SandboxIR/SandboxIRValues.def"],
```

Comment:
The .def file is just a bunch of X macros so it should probably go in `textual_hdrs`

https://bazel.build/reference/be/c-cpp#cc_library.textual_hdrs:
> This is the location for declaring header files that cannot be compiled on their own; that is, they always need to be textually included by other source files to build valid code.

Reply:
Done